src/main/java/com/yurjinia/user/service/UserProfileService.java

```diff
@@ -66,6 +68,10 @@ public UserProfileDTO getUserProfile(String userEmail) {
     }
 
     public void updateAvatar(String userEmail, MultipartFile image) {
+        if (Objects.isNull(image)) {
+            return;
```

Comment:
throw an exception in case there is no provided avatar, create a separate validation method.